src/util/sensitive_string.js

```diff
@@ -19,14 +19,19 @@ class SensitiveString {
         } else {
             throw new TypeError(`SensitiveString should be a string, got ${type}`);
         }
+        if (process.env.DISABLE_SENSITIVE_STRING === 'true') {
+            this.sensitive_val = this.val;
```

Comment:
didn't we said we'll make it:
this.sensitive_val = 'SENSITIVE-' + this.val;
or you decided different in the end?

Reply:
I thought about it again and it seemed unnecessary and annoying. no problem to add it later if we want